editor/editor_inspector.cpp

```diff
@@ -206,9 +207,27 @@ void EditorProperty::_notification(int p_what) {
 					}
 				}
 
+				bool show_config_info_icon = !config_info.is_empty() && !read_only;
+				if (show_config_info_icon) {
+					Ref<Texture2D> config_info_icon;
+
+					String severity = EditorConfigurationInfo::get_max_severity(config_info);
```

Comment:
```suggestion
					const String severity = EditorConfigurationInfo::get_max_severity(config_info);
```
Same with other Strings.